Remove inappropriate "please", minor rewording (#4249)
iterative/dvc.org: content/docs/user-guide/data-management/managing-external-data.md

```

⚠️ DVC requires both SSH and SFTP access to work with remote SSH locations.

> Don't forget to check that you are able to connect both ways with tools like `ssh` and

Reviewer: ```suggestion
Check that you can connect both ways with tools like `ssh` and
```

custom cache location for local paths outside of your project.

> > Except for external data on different storage devices or partitions mounted on

Reviewer: ```suggestion
> Exceptions to the above include: external data on different storage devices, and
partitions mounted on
```

> > the same file system (e.g. `/mnt/raid/data`). In that case you can setup an
> > external cache in that same drive to enable [file links] and avoid copying

Reviewer: ```suggestion
> the same file system (e.g. `/mnt/raid/data`). In such cases, set up an
> external cache in the same drive to enable [file links] and avoid copying
```
